.github/workflows/lint-php.yml

```
@@ -0,0 +1,65 @@
+name: Lint PHP
+
+on:
+  workflow_call:
+    inputs:
+      PHP_MATRIX:
+        description: Matrix of PHP versions as a JSON formatted object.
+        default: '["8.0"]'
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer.
+        default: '--prefer-dist'
+        required: false
+        type: string
+      LINT_ARGS:
+        description: Set of arguments passed to PHP-Parallel-Lint.
```

No dashes in the name of the tool according to https://github.com/php-parallel-lint/PHP-Parallel-Lint.
```suggestion
        description: Set of arguments passed to PHP Parallel Lint.
```